homeassistant/components/sensor/opensky.py

```diff
@@ -0,0 +1,141 @@
+"""
+Sensor for the Open Sky Network.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.opensky/
+"""
+import logging
+from datetime import timedelta
+
+import requests
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_LATITUDE, CONF_LONGITUDE,
+    ATTR_ATTRIBUTION, ATTR_LATITUDE, ATTR_LONGITUDE,
+    LENGTH_KILOMETERS, LENGTH_METERS)
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import distance as util_distance
+from homeassistant.util import location as util_location
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=12)  # opensky public limit is 10 seconds
```

Comment:
Well, if you want to know when a flight enters your region, it'll be gone already in 30 seconds by the time the sensor updates.

But it is a `SCAN_INTERVAL`, so people with that use case could always set it lower if we default to 30.